Makefile

```diff
@@ -0,0 +1,56 @@
+.DEFAULT_GOAL := help
+SHELL := /bin/bash
+VENV = venv
+
+OK := $(shell if [ -d $(VENV) ]; then echo "ok"; fi)
+
+
+
+ifeq ($(OK),)
+	path := $(shell which python3)
+	PYTHON := $(path)
+	PIP := pip
+else
+	PYTHON = $(VENV)/bin/python3
+	PIP = $(VENV)/bin/pip
+endif
+
+b:
+	@echo $(OK)
+	@echo $(PYTHON)
+	@echo $(PIP)
+
+install: 
+	$(PIP) install --upgrade pip
+	$(PIP) install pipx nox
+	$(PIP) install '.[$(if $(collection),$(collection),all)]'
+	$(PIP) install .[tests]
+
+activate: 
+	. $(VENV)/bin/activate
+	
+init_venv:
+	test -d venv || $(PYTHON) -m venv $(VENV)
+	. $(VENV)/bin/activate
```

Comment:
Should venv be an env variable? Can we make it an argument?

e.g. `make activate --venv dev`